packages/admin/admin-rte/src/core/translation/stateToHtml.ts

```diff
@@ -0,0 +1,39 @@
+import { EditorState } from "draft-js";
+import { stateToHTML } from "draft-js-export-html";
+
+import { IRteOptions } from "../Rte";
+
+export function stateToHtml({ editorState, options }: { editorState: EditorState; options: IRteOptions }) {
+    const contentState = editorState.getCurrentContent();
+
+    const customInlineStyleKeys = options?.customInlineStyles ? Object.keys(options.customInlineStyles) : [];
+
+    const inlineStyles: any = {
+        SUB: { element: "sub" },
+        SUP: { element: "sup" },
+    };
+
+    customInlineStyleKeys.forEach((item) => {
+        inlineStyles[item] = { element: "span", attributes: { style: [item] } };
+    });
+
+    const linkDataList: { id: string; data: any }[] = [];
```

Comment:
Please rename to entities, we might support other entities in the future

Reply:
0bbd007a205bd1dbb1d7049a0e3aa78072663c2e